tests/cbor_decode/test3_simple/src/main.c

```diff
@@ -121,6 +121,289 @@ uint8_t serial_rec_input2[] = {
 	0xfb
 };
 
+uint8_t crash_vector0[] = {
```

Comment:
Can we add a comment on these to describe where they come from?

Reply:
Added

Reply:
Same thing here, can't find comment in diff.

Reply:
Sorry, I added the comment to the function that uses the vectors.

Reply:
Is that the one on line 439?

> /* Run test cases discovered via fuzzing. */